src/server/JasmineGraphServer.cpp

```diff
@@ -2360,6 +2362,1019 @@ int JasmineGraphServer::removePartitionThroughService(string host, int port, str
+        serv_addr.sin_port = htons(port);
+        if (connect(sockfd, (struct sockaddr *) &serv_addr, sizeof(serv_addr)) < 0) {
+            std::cerr << "ERROR connecting" << std::endl;
+            //TODO::exit
+        }
+
+        bzero(data, 301);
+        int result_wr = write(sockfd, JasmineGraphInstanceProtocol::HANDSHAKE.c_str(), JasmineGraphInstanceProtocol::HANDSHAKE.size());
+
+        if(result_wr < 0) {
+            server_logger.log("Error writing to socket", "error");
+        }
+
+        server_logger.log("Sent : " + JasmineGraphInstanceProtocol::HANDSHAKE, "info");
+        bzero(data, 301);
+        read(sockfd, data, 300);
+        string response = (data);
+
+        response = utils.trim_copy(response, " \f\n\r\t\v");
+
+        while (!loop) {
+            if (response.compare(JasmineGraphInstanceProtocol::HANDSHAKE_OK) == 0) {
+                server_logger.log("Received : " + JasmineGraphInstanceProtocol::HANDSHAKE_OK, "info");
+                result_wr = write(sockfd, masterIP.c_str(), masterIP.size());
+
+                if(result_wr < 0) {
+                    server_logger.log("Error writing to socket", "error");
+                }
+
+                server_logger.log("Sent : " + masterIP, "info");
+                bzero(data, 301);
+                read(sockfd, data, 300);
+                response = (data);
+
+                if (response.compare(JasmineGraphInstanceProtocol::HOST_OK) == 0) {
+                    server_logger.log("Received : " + JasmineGraphInstanceProtocol::HOST_OK, "info");
+                } else {
+                    server_logger.log("Received : " + response, "error");
+                }
+
+                bzero(data, 301);
+                read(sockfd, data, 300);
+                response = (data);
+                response = utils.trim_copy(response, " \f\n\r\t\v");
+
+                if (response.compare(JasmineGraphInstanceProtocol::OK) == 0) {
+                    server_logger.log("Received : " + JasmineGraphInstanceProtocol::OK, "info");
+                    result_wr = write(sockfd, graphID.c_str(), graphID.size());
+
+                    if(result_wr < 0) {
+                        server_logger.log("Error writing to socket", "error");
+                    }
+
+                    server_logger.log("Sent : Graph ID " + graphID, "info");
+
+                    JasmineGraphServer::collectBloomFiltersToMaster(coordinatorHost, workerPort, designatedWorkerHost,
+                                                designatedWorkerPort, designatedWorkerPort, graphID, 4, masterIP);
+                }
+
+            }
+
+
+        }
+    }
+}
+
+int JasmineGraphServer::signalOrganizations(string organizationHost, string designatedWorkerHost, string designatedWorkerPort, string graphID, string masterIP) {
+    Utils utils;
+    Logger instance_logger;
+    bool result = true;
+    std::cout << pthread_self() << " host : " << organizationHost << "port : " << 7777 << std::endl;
```

Comment:
Shall we remove this cout statement? I do not see any use of it.

Reply:
It  is used for server logging purposes in every scenario where a socket connection is created